src/app/containers/App/App.jsx

```diff
@@ -82,12 +85,15 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
         });
       });
 
-      getInitialData({
-        path: location.pathname,
-        service: nextService,
-        variant: nextVariant,
-        pageType: nextPageType,
-      }).then(data => {
+      const updateRemoteConfig = async () => {
+        const nextRemoteConfig = await getRemoteConfig(nextService);
```

Comment:
When using async you can pretty much drop the use of `.then`. The loaderPromise code though I don't entirely understand so I didn't want to touch it too much. I think it could use a bit of a refactor for sure.

Reply:
In the swarm review regarding this PR, it was decided that the work to refactor this may take place outside of this PR. Please refer to the Simorgh tech spreadsheet and back to this PR for more details later.